api/openstorage.yaml

```diff
@@ -774,17 +784,19 @@ paths:
                      "secure_device_path": "",
                      "background_processing": false
                     }]
-    post:
-      summary: Create New Volume
-      description: Create new volume according to spec
-      parameters:
-        - name: spec
-          in: body
+      post:
+        summary: Create New Volume
+        description: Create new volume according to spec
```

Comment:
There should be examples to create volume of specific size, replication factor, .. pretty much everything that we allow.